src/scheduler/pbs_holidays

```diff
@@ -34,44 +34,25 @@
  * "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
  * trademark licensing policies.
 *
-
 *
-YEAR  2018
+* UNCOMMENT AND CHANGE THIS TO THE CURRENT YEAR
+*YEAR  1970
```

Comment:
As I said in the forum post, use '#' to comment out the lines.  The '#'  always worked, but '*' was the correct comment character for this file to be compliant with UNICOS.

Reply:
Ok, I'll make it '#' and keep the check for '*' for backward compatibility.

btw, I actually don't see your comment in the forum, could you please post it once more? Thanks.